backend/src/test/java/com/woowacourse/gongcheck/acceptance/DatabaseInitializer.java

```diff
@@ -0,0 +1,80 @@
+package com.woowacourse.gongcheck.acceptance;
+
+import java.sql.DatabaseMetaData;
+import java.sql.ResultSet;
+import java.util.ArrayList;
+import java.util.List;
+import javax.persistence.EntityManager;
+import javax.sql.DataSource;
+import org.springframework.beans.factory.InitializingBean;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import org.springframework.transaction.annotation.Transactional;
+
+@Component
+public class DatabaseInitializer implements InitializingBean {
+
+    @Autowired
+    private EntityManager entityManager;
+
+    @Autowired
+    private DataSource dataSource;
+
+    private List<String> tableNames = new ArrayList<>();
+
+    @Override
+    public void afterPropertiesSet() {
+        try {
+            DatabaseMetaData metaData = dataSource.getConnection().getMetaData();
+            ResultSet tables = metaData.getTables(null, null, null, new String[]{"TABLE"});
+            while (tables.next()) {
+                String tableName = tables.getString("TABLE_NAME");
+                tableNames.add(tableName);
+            }
+        } catch (Exception e) {
+            throw new RuntimeException();
```

Comment:
```suggestion
            throw new RuntimeException("테이블 이름 세팅에 실패하였습니다.");
```
라고 에러메시지를 구체화하면 좋을 것 같아요